docs/src/warthog-cli/codegen.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I'm not sure the CLI needs sub-pages.  The only command that is complex is `generate`, so it could probably all be on one page for now.